pages/docs/concepts/asyncapi-document/adding-channels.md

```diff
@@ -0,0 +1,179 @@
+    s1(Server 1)
+    s2(Server 2)
+    s3(Server 3)
+  end
+
+  subgraph Channel
+    ch(Channel)
+  end
+
+  style ch fill:#47BCEE,stroke:#47BCEE;
+
+  c1 --- ch
+  c2 --- ch
+  c3 --- ch
+
+  s1 --- ch
+  s2 --- ch
+  s3 --- ch
+```
+
+In the above flowchart, messages published to the channel to reach all connected clients, regardless of their server connection.
+
+Here is an example of how you might specify that a channel is available only on specific servers:
+
+```yml
+channels:
+  userSignedUp:
+    address: user/signedup
+    messages:
+      userSignedUp:
+        description: An event describing that a user just signed up.
+servers:
+  serverA:
+    url: serverA.example.com
+  serverB:
+    url: serverB.example.com
+```
+
+The above code defines a channel `user/signedup` for subscribing to user signup events and two servers, `Server A` and `Server B`, with their respective URLs, enabling clients to receive user signup notifications from either server.
+
+Specifying a channel and servers allows the AsyncAPI document to clarify where each channel can be found. This method is particularly beneficial in complex systems where different servers might be responsible for different operations.
+
+### Multiple Channels in Single Server
+
+Having multiple channels in one server allows for better organization and management of different events or messages within the same server instance, providing a more modular and scalable architecture for handling various functionalities. It enables clients to send to specific channels based on their interests or requirements, ensuring efficient message delivery and reducing complexity in the overall system design.
+
+Here is a diagram showing a single server hosting multiple channels:
+
+```mermaid
+graph TD
+  subgraph Server
+    s1(Server)
+  end
+
+  subgraph Channels
+    c1{Channel 1}
+    c2{Channel 2}
+    c3{Channel 3}
+  end
+
+  style c1 fill:#47BCEE,stroke:#47BCEE;
+  style c2 fill:#47BCEE,stroke:#47BCEE;
+  style c3 fill:#47BCEE,stroke:#47BCEE;
+
+  s1 --> c1
+  s1 --> c2
+  s1 --> c3
+```
+
+Hosting multiple channels in single server allows user to organize and manage better in case for multiple events or messages by facilitating modular and scalable system architecture.
+
+Here is an example of multiple channels available on specific servers:
+
+```yml
+channels:
+  userSignedUp:
+    address: user/signedup
+    messages:
+      userSignedUp:
+        description: An event describing that a user just signed up.
+  userActivated:
+    address: user/activated
+    messages:
+      userActivated:
+        description: An event describing that user activation events.      
+servers:
+  serverA:
+    url: serverA.example.com
+```
+
+The YAML code defines multiple channels, `user/signedup` and `user/activated`, within the `serverA` server, enabling clients to subscribe and receive user signup and activation events.
+
+To know more details on how to add server check [add servers](add-server.md).
```

Comment:
same as above, this link 404s and must be fixed

Reply:
Here I have referred to this PR - https://github.com/asyncapi/website/pull/1927
So link 404 will be fixed after it is merged.